internal/lint/subscribe.go

```diff
@@ -39,10 +41,25 @@ func Subscriber(input *SubscriberInput) {
 			bps.RawSubMessage("message-2"),
 		})
 		handler = &mockHandler{}
+
+		// give consumer only 1s per test:
+		ctx, cancel = context.WithTimeout(context.Background(), time.Second)
+	})
+
+	ginkgo.AfterEach(func() {
+		cancel()
 	})
 
 	ginkgo.It("should subscribe", func() {
-		Ω.Expect(subject.Subscribe(ctx, topic, handler)).To(Ω.Succeed())
+		err := subject.Subscribe(ctx, topic, handler)
+
+		// subscriber implementations are allowed to:
+		// - either return immediately
+		// - or block until "unsubscribed" (deadline exceeded for this test suite)
+		if err != nil {
+			Ω.Expect(err).To(Ω.MatchError(context.DeadlineExceeded))
+		}
```

Comment:
This approach is ugly, any suggestions?

Previously, it was easy to expect success because of non-blocking InMem/File subscribers.
But kafka blocks after receiving all the seeded messages (expected).